components/organisms/Education.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import {TimelineEvent} from '../molecules/Timeline';
+import {BookOpenIcon} from '@heroicons/react/24/solid';
+
+
+interface EducationEntry {
+    name: string;
+    startDate: string;
+    endDate: string;
+    description?: string;
+    icon?: React.ReactNode;
+}
+
+const education: EducationEntry[] = [
+    {
+        name: 'Federal State Budget-Financed Educational Institution of Higher Education The Bonch-Bruevich',
+        startDate: '2022',
+        endDate: '2026',
+        description: 'Study at the university in the direction of "Technology".',
+        icon: <BookOpenIcon width={36} />,
+    },
+    {
+        name: 'Lyceum No. 5',
+        startDate: '2012',
+        endDate: '2022',
+        description: 'Secondary education with an emphasis in technical sciences.',
+        icon: <BookOpenIcon width={36} />,
+    }
+];
+
+export default function Education() {
+    const currentDate = new Date().getFullYear();
+
+    return (
+        <div className="md:container text-muted-foreground self-center pt-10 w-full relative">
+            {education.map((school, index) => (
+                <TimelineEvent key={`timeline-education-${index}`} active={!school.endDate || parseInt(school.endDate) >= currentDate}
```

Comment:
```suggestion
                <TimelineEvent key={`timeline-education-${index}`} active={!school.endDate || parseInt(school.endDate) >= currentDate}
```

Чтоб не делать преобразования строки в число можно использовать число сразу =)